hw/top_darjeeling/data/top_darjeeling.hjson

```diff
@@ -255,9 +256,10 @@
       clock_srcs: {clk_i: "io_div4", clk_kmac_i: "main"},
       clock_group: "secure",
       reset_connections: {rst_ni: "lc_io_div4", rst_kmac_ni: "lc"},
-      base_addr: {
-        hart: "0x30140000",
-      },
+      base_addrs: {
+        core: {hart: "0x30140000"},
+        dmi: {soc_lc_ctrl_dmi: "0x0"},
```

Comment:
I think another approach would be to have the LC control also hook up to the existing XBAR that connects to the JTAG mailbox and the socdbgctrl currently
The XBAR will look something like this
3 host ports (upstream DV environment currently requires 3 different host interfaces - one for LC_CTRL, RV_DM and socdbgctrl). These host ports will be connected to the respective JTAG drivers in chip wrapper. 
4 device ports
1. --> RV_DM   --> uses the main clock
2. --> LC_CTRL --> uses the LC_CRTL clokc (io_div_4)
3. --> JTAG mailbox --> clk TBD
4. --> socdbgctrl --> io_div_4 clock
the host port to the device port connectivity is manages in the XBAR
This is prevent the need to create yet another address space

Reply:
I do not think there should be a security implication here. All transactions are ultimately funnelled via the same JTAG controller